src/content/dataSubmissions/DataSubmission.tsx

```diff
@@ -200,7 +200,7 @@ const StyledFlowContainer = styled(Box)({
 });
 
 const URLTabs = {
-  DATA_ACTIVITY: "data-activity",
+  UPLOAD_ACTIVITY: "upload-activity",
```

Comment:
I forgot to mention this, but we also need to update the link to this page on the Data Submission List page. It still points to `[...]/data-activity`. I believe that's the only other place that needs updated

https://github.com/CBIIT/crdc-datahub-ui/blob/890410acabfa6fe85a093507b3185fb57bfbb2c5/src/content/dataSubmissions/DataSubmissionsListView.tsx#L134